### PR TITLE
Allows country-template to pass check-style makefile command

### PR DESCRIPTION
**Pull request by verbman**
Allows country-template to pass check-style makefile command out of the box

* Technical improvement.
* Impacted periods: none.
* Impacted areas: all
* Details:
  - If you follow the make new country template instructions and at the end run `make test` there are style errors. This resolves those errors for that satisfying green tick.

- - - -

These changes:

- Fix or improve an already existing calculation.
- Change non-functional parts of this repository (for instance editing the README)

### CODE REVIEW

**Review by MattiSG — APPROVED** [review]
Thanks @verbman for the fix and for the typo correction 🙏

**Comment by MattiSG** [comment]
Weird, the CI [says](https://github.com/openfisca/country-template/actions/runs/4825467842/jobs/8596363421?pr=130) now that `Run make check-style` does not pass 🤔 Do you intend to fix those errors as well in this PR? 🙂

**Comment by verbman** [comment]
To describe the problem as it first presented, here's the github lint failure and the opposite local lint failure:
```
openfisca_country_template/__init__.py:14:1: I201 Missing newline between import groups. 'from openfisca_country_template import entities' is identified as Application and 'from openfisca_core.taxbenefitsystems import TaxBenefitSystem' is identified as Third Party.
```
If I add that newline back in and run it locally I get the issue (which the PR was attempting to resolve):
```
openfisca_cov/__init__.py:15:1: I202 Additional newline in a group of imports. 'from openfisca_cov import entities' is identified as Third Party and 'from openfisca_core.taxbenefitsystems import TaxBenefitSystem' is identified as Third Party.
```
I now understand what the issue is and it makes this PR redundant so I will close it.
The way to resolve this issue will be something along the lines of editing `bootstrap.sh` [to include `setup.cfg` say on line 39](https://github.com/openfisca/country-template/blob/master/bootstrap.sh#LL39C1-L39C152):

```
sed -i.template "s|country_template|$lowercase_country_name|g" README.md setup.py setup.cfg .github/workflows/workflow.yml Makefile MANIFEST.in $all_module_files
```
In other words, the setup.cfg file is the source of the local linting issue because in the renamed repository it still references `openfisca_country_template` on lines 17,18 and 36

**Comment by verbman** [comment]
Raised as new [issue 131](https://github.com/openfisca/country-template/issues/131)
